package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbon-react",
-  "version": "68.2.0",
+  "version": "68.3.0",
```

Comment:
Has something happened with your git history, looks like you've got some release data in here?

Reply:
Same as previous